ReactNativeClient/lib/joplin-renderer/MdToHtml/rules/audio.js

```diff
@@ -0,0 +1,27 @@
+// const Resource = require('lib/models/Resource.js');
+const utils = require('../../utils');
+
+function installRule(markdownIt, mdOptions, ruleOptions) {
+	const defaultRender = markdownIt.renderer.rules.link_open;
+
+	markdownIt.renderer.rules.link_open = (tokens, idx, options, env, self) => {
+		const Resource = ruleOptions.ResourceModel;
+
+		const token = tokens[idx];
+		const src = utils.getAttr(token.attrs, 'href');
+
+		if (!Resource.isResourceUrl(src) || ruleOptions.plainResourceRendering) return defaultRender(tokens, idx, options, env, self);
+
+		const r = utils.imageReplacement(ruleOptions.ResourceModel, src, ruleOptions.resources, ruleOptions.resourceBaseUrl);
```

Comment:
Feels wrong to call imageReplacement here, this function should probably be generalized to resourceReplacement.

Reply:
Scratch that, I had a misunderstanding about what this does,

Reply:
Actually, I think you were right at first. I had actually changed it to resourceReplacement initially and I can't remember why I changed it back.